package.json

```diff
@@ -105,7 +106,7 @@
 		"electron": "^32.2.7"
 	},
 	"engines": {
-		"node": ">=20.9.0 <21 || >=22"
+		"node": ">=20.18.1 <21 || 22 || >=23"
```

Comment:
can you change this back to >=22 at the end? Thx.

Reply:
But... engines.node is not fixed on package-lock

Reply:
how I am I supposed to know all the steps?  I don't know what is is the lockfile except it always causes trouble..
posted an update but too late

Reply:
will update dependencies before release and fix it ... I should not merge late at night ...